clang-tools-extra/clang-tidy/ClangTidy.cpp

```diff
@@ -353,7 +356,8 @@ ClangTidyASTConsumerFactory::ClangTidyASTConsumerFactory(
   if (Context.canExperimentalCustomChecks() && custom::RegisterCustomChecks)
     custom::RegisterCustomChecks(Context.getOptions(), *CheckFactories);
 #endif
-  for (ClangTidyModuleRegistry::entry E : ClangTidyModuleRegistry::entries()) {
+  for (ClangTidyModuleRegistry::entry const E :
```

Comment:
```suggestion
  for (const ClangTidyModuleRegistry::entry E :
```

But shouldn't clang-format be able to police const placement? https://clang.llvm.org/docs/ClangFormatStyleOptions.html#qualifierorder

Reply:
We use https://clang.llvm.org/docs/ClangFormatStyleOptions.html#qualifieralignment but I guess it is edge case that wasn't handled?

Reply:
It was issue of clang-format-21: https://github.com/llvm/llvm-project/issues/167121
Now I ran clang-format-22: https://github.com/llvm/llvm-project/pull/167122